labelbox/schema/dataset.py

```diff
@@ -68,8 +68,8 @@ class Dataset(DbObject, Updateable, Deletable):
 
     def data_rows(
         self,
-        from_cursor: str = None,
-        where: Comparison = None,
+        from_cursor: Optional[str] = None,
```

Comment:
this is tricky, but correct change I believe, based on https://docs.python.org/3/library/typing.html#typing.Optional